parachain/src/chain_spec.rs

```diff
@@ -613,21 +620,21 @@ fn kintsugi_mainnet_genesis(
         vault_registry: kintsugi_runtime::VaultRegistryConfig {
             minimum_collateral_vault: vec![(CurrencyId::KSM, 0)],
             punishment_delay: kintsugi_runtime::DAYS,
-            system_collateral_ceiling: vec![(default_pair(CurrencyId::KSM), 317 * CurrencyId::KSM.one())], /* 317 ksm, about 100k
-                                                                                                            * USD at
-                                                                                                            * time of writing */
+            system_collateral_ceiling: vec![(default_pair_kintsugi(CurrencyId::KSM), 317 * CurrencyId::KSM.one())], /* 317 ksm, about 100k
+                                                                                                                     * USD at
+                                                                                                                     * time of writing */
             secure_collateral_threshold: vec![(
-                default_pair(CurrencyId::KSM),
-                FixedU128::checked_from_rational(260, 100).unwrap(),
-            )], /* 260% */
+                default_pair_kintsugi(CurrencyId::KSM),
```

Comment:
@savudani8 @gregdhill why are the thresholds so heavily increased? We need to discuss these parameter changes before making changes. There is a risk/product trade-off here. Increasing the collateralization by 100% points is reducing the profitability of running a vault significantly.